.changeset/violet-mirrors-compare.md

```diff
@@ -0,0 +1,6 @@
+---
+"@mds-core/mds-policy-service": minor
+"@mds-core/mds-types": minor
```

Comment:
```suggestion
"@mds-core/mds-policy-service": patch
"@mds-core/mds-types": patch
```
No interface additions or changes to exported types? Feels like patch.

Reply:
I didn't notice it making patch level changes an option.